lib/citgm.js

```diff
@@ -120,11 +121,11 @@ Tester.prototype.run = function() {
         self.emit('fail', err);
         payload.error = err;
       }
-      if (self.testOutput !== '') {
-        payload.testOutput += self.testOutput + '\n';
+      if (self.testOutput.length) {
+        payload.testOutput += self.testOutput.toString() + '\n';
       }
-      if (self.testError !== '') {
-        payload.testOutput += self.testError + '\n';
+      if (self.testError.length) {
+        payload.testOutput += self.testError.toString() + '\n';
```

Comment:
so in doing this we are still keeping the majority of the payload as a string inside the main object while the application is running. We could potentially move the `toString()` to later in the application, likely in the reporter.

Thoughts?